.github/workflows/jreleaser.yml

```diff
@@ -29,6 +37,12 @@ jobs:
           JRELEASER_NEXUS2_MAVEN_CENTRAL_USERNAME: ${{ secrets.JRELEASER_NEXUS2_MAVEN_CENTRAL_USERNAME }}
           JRELEASER_NEXUS2_MAVEN_CENTRAL_PASSWORD: ${{ secrets.JRELEASER_NEXUS2_MAVEN_CENTRAL_PASSWORD }}
 
+      - name: Sign artifacts with sigstore/cosign
+        if: inputs.attest-artifacts
+        uses: actions/attest-build-provenance@1c608d11d69870c2092266b3f9a6f3abbf17002c # v1.4.3
+        with:
+          subject-path: './target/staging-deploy/**/*.jar'
```

Comment:
`push-to-registry` should also be included, right? Based on the [README](https://github.com/actions/attest-build-provenance), it seems this would not be pushed anywhere. However, I am unsure what the value would be here.

Reply:
https://github.com/INRIA/spoon/pull/6024 is pushing without this so need to accept this suggestion.

Reply:
@RafDevX could you find out what `push-to-registry` means?

Reply:
As I understand it's only used to push the attestation to image registries, if applicable, in addition to the GitHub sigstore. Explained more in detail in this section of [README](https://github.com/actions/attest-build-provenance?tab=readme-ov-file#container-image)